scripts/docker_build.sh

```diff
@@ -0,0 +1,3 @@
+
+#!/bin/sh
+docker build -f docker/Dockerfile.builder -t sasl_auth:latest .
```

Comment:
I fixed that up, but I'm still not terribly sure we want this Dockerfile to be here. Thoughts?

Reply:
I finally managed to get all tests passing after doing the following:

```
sudo kadmin.local
Authenticating as principal root/admin@EXAMPLE.COM with password.
kadmin.local:  addprinc -randkey kafka/localhost
No policy specified for kafka/localhost@EXAMPLE.COM; defaulting to no policy
Principal "kafka/localhost@EXAMPLE.COM" created.
kadmin.local:  ktadd  kafka/localhost
Entry for principal kafka/localhost with kvno 2, encryption type aes256-cts-hmac-sha1-96 added to keytab FILE:/etc/krb5.keytab.
Entry for principal kafka/localhost with kvno 2, encryption type aes128-cts-hmac-sha1-96 added to keytab FILE:/etc/krb5.keytab.
kadmin.local:  q
```
I think it is worth having all setup needed to run the tests automated so that one can run the tests with a single command. I have created an issue for that here so we do not need to do this in this PR: https://github.com/kafka4beam/sasl_auth/issues/11

Reply:
I created another issue for fixing the instructions https://github.com/kafka4beam/sasl_auth/issues/12 so I think we do not need to fix them before merging this PR.